### PR TITLE
Update file-integrity-operator enhancement

### PR DESCRIPTION
**Pull request by mrogers950**
This PR updates the enhancement proposal with some more details about the current design as well as extending the roadmap section, re: https://github.com/openshift/enhancements/pull/79#issuecomment-576396370 
@smarterclayton @cgwalters @ashcrow @JAORMX @jhrozek

### CODE REVIEW

File: enhancements/security/file-integrity-operator.md [hunk 1]
```diff
@@ -103,10 +117,23 @@ The operator is intended to be the sole controller of its operand resources (con
...
+* Further secure AIDE process containers with SELinux to only allow writing to the DB and log files.
+* Enable the use of object storage for exposing the AIDE logs instead of configMaps. See the Drawbacks section for more regarding AIDE log storage.
+* Add a "Provider" field to the API and enable other integrity checking mechanisms.
+  * Enable non-userspace and strongly attested integrity checking mechanism(s). Some suggested candidates include fs-verity and IMA. Note: Both IMA/fs-verity methods provide more integrity assurance over AIDE _only_ when backed by a TPM.
+    * fs-verity: According to https://www.kernel.org/doc/html/latest/filesystems/fsverity.html, fs-verity works only on read-only files. Requires ext4 formatting options and block size to be == PAGE_SIZE. Files need to be ioctl'ed as verity files and have signature verification performed in userspace. Unless fs-verity is backed by attested dm-verity volumes (not feasible for nodes) or tied into IMA somehow, this provider seems insufficient for our use.
```
Comment (cgwalters) [inline enhancements/security/file-integrity-operator.md]: Well, we'll have to have a debate about that :smile:
But it doesn't have to happen now.

Comment (mrogers950) [reply enhancements/security/file-integrity-operator.md]: No problem :) 
This was just my general assessment after reading the documentation. If there's ultimately a way we can leverage fs-verity (perhaps only together with IMA ?) that is all good with me.

**Comment by ashcrow** [comment]
If there are no objections or added questions by tomorrow morning I'm planning on merging this update.

**Pull request merged**